lib/util/esWrapper.js

```diff
@@ -225,6 +225,15 @@ class ESWrapper {
         : error.meta.body.error.reason;
     }
 
+    if (error.meta.body && error.meta.body.error) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)